bin/storm.py

```diff
@@ -263,6 +263,34 @@ def upload_credentials(*args):
         jvmtype="-client",
         extrajars=[USER_CONF_DIR, STORM_BIN_DIR])
 
+def blobstore(*args):
+    """Syntax: [storm blobstore [cmd]]
+
+    list [KEY...] - lists blobs currently in the blob store
+    cat [-f FILE] KEY - read a blob and then either write it to a file, or STDOUT (requires read access).
+    create [-f FILE] [-a ACL ...] [--repl-fctr NUMBER] KEY - create a new blob. Contents comes from a FILE
+         or STDIN. ACL is in the form [uo]:[username]:[r-][w-][a-] can be comma
+         separated list.
```

Comment:
Documentation suggestions:

It might be good to add a brief example here of an ACL, as we have the same for the replication factor.  Technically, the Access Control List is a list of entries, each of the form `<u:USERNAME:[r-][w-][a-] | o::[r-][w-][a-]>[,...]`

`For example, to allow everyone read access and to allow alice read, write, and admin access, the ACL would have two entries: "o::r,u:alice:rwa"`

Since there are multiple commands that take ACL option arguments, we might want to just add some "ACL Example" to the end of the help text.